docs/src/templates/pageDemo.jsx

```diff
@@ -1159,4 +1179,11 @@ const mapStateToProps = (state) => {
     };
 };
```

Comment:
Should we move `mapStateToProps()` up to the top, below the `propTypes` and `defaultProps` declarations while we're working in here?